data_steward/tools/copy_dataset_to_output_prod.py

```diff
@@ -145,17 +146,17 @@ def get_arg_parser() -> argparse.ArgumentParser:
     )
     dataset_object = bq.define_dataset(args.output_prod_project_id,
                                        output_dataset_name, description, labels)
-    client.create_dataset(dataset_object, exists_ok=False)
+    bq_client.create_dataset(dataset_object, exists_ok=False)
 
     #Copy tables from source to destination
     LOGGER.info(
         f'Copying tables from dataset {args.src_project_id}.{args.src_dataset_id} to {args.output_prod_project_id}.{output_dataset_name}...'
     )
-    bq.copy_datasets(client, f'{args.src_project_id}.{args.src_dataset_id}',
+    bq.copy_datasets(bq_client, f'{args.src_project_id}.{args.src_dataset_id}',
                      f'{args.output_prod_project_id}.{output_dataset_name}')
 
     #Append extra columns to person table
     LOGGER.info(f'Appending extract columns to the person table...')
-    update_person(client, args.output_prod_project_id, output_dataset_name)
+    update_person(bq_client, output_dataset_name)
```

Comment:
This is good to highlight. We don't have to rely on the project being passed as long as the client is instantiated with it.